### PR TITLE
feat: Firefox Smart Keywords implementation, closes #34

### LINKED ISSUE #34: Having Firefox recognize search field
On Firefox, on most websites, it's possible to right click the search field and select "Add a Keyword for this search". Sites like IMDB, Youtube, Github etc. 
screenshot from 2019-01-29 18 14 31 2x

This enables me to more start typing a search term right from my url bar:
screenshot from 2019-01-29 18 20 09 2x

More info: https://support.mozilla.org/en-US/kb/how-search-from-address-bar

I think this could be enabled by:
1. Allowing searching using a query parameter (`?q=searchterm`). Currently searching only works using: `/apps/search/searchterm`. (I understand this is a big ask)
2. Adding a `<form action="/apps/">` element around the input field. This will make Firefox recognize it as a search. 
3. Giving the input the name of the query argument `name="q"` for example.

### PR DESCRIPTION
**Pull request by yungmides**
Hello 👋 

This PR addresses the #34 issue, by allowing Firefox users to right-click on the search input to "Add A Keyword for this Search...".

### CODE REVIEW

**Review by antfu — CHANGES_REQUESTED** [review]
Can you revert the formatting changes? Thanks

**Comment by yungmides** [comment]
Done ! Lmk if you need anything else 👍

File: src/components/IconSet.vue [hunk 1]
```diff
@@ -142,7 +154,10 @@
           @click="selectingMode = false"
         >
           Selecting Mode
-          <Icon icon="carbon:close" class="inline-block text-xl align-text-bottom" />
+          <Icon
+            icon="carbon:close"
+            class="inline-block text-xl align-text-bottom"
+          />
```
Comment (antfu) [inline src/components/IconSet.vue:160]: I mean, most of the changes in this file are formatting changes, can you revert them so the PR would be clear on fixing only the necessary part

Comment (yungmides) [reply src/components/IconSet.vue:160]: oh, got it ! Sorry for the confusion, working on it

**Pull request merged**